### PR TITLE
Mocked token tests.

### PR DESCRIPTION
**Pull request by catenocrypt**
* 'Token' newman tests (for 9 coins) changed to work with mocks.
* Token tests added to mocked tests executed in CI.
* Mocked test description added to README.

### CODE REVIEW

File: mock/ext-api-dyson/get/callisto-api-commands1.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+module.exports = {
+    path: '/callisto-api/:command1?',
+    template: function(params, query, body) {
+        //console.log(query);
+        switch (params.command1) {
+            case 'transactions':
+                if (query.address === '0x3083a7ec44ca2b038d4be4b0798152f948f0f3d7') {
+                    return JSON.parse(`
+                        {
```
Comment (vikmeup) [inline mock/ext-api-dyson/get/callisto-api-commands1.js:18]: did you consider storing these JSON responses in files?

File: mock/ext-api-dyson/get/gochain-api-commands1.js [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+/// See:
+/// curl "http://localhost:3000/gochain-api/transactions?address=0xEd7F2e81B0264177e0df8f275f97Fd74Fa51A896"
+/// curl "http://localhost:3000/gochain-api/tokens?address=0x0Fd98FB42C439E5F6484f7E71Caa6661d81d0628"
+/// curl "https://{go rpc}/transactions?address=0xEd7F2e81B0264177e0df8f275f97Fd74Fa51A896"
+/// curl "https://{go rpc}/ 
+/// curl "http://localhost:8420/v1/gochain/0xEd7F2e81B0264177e0df8f275f97Fd74Fa51A896"
+/// curl "http://localhost:8420/v2/gochain/tokens/0x0Fd98FB42C439E5F6484f7E71Caa6661d81d0628?Authorization=Bearer"
+
+module.exports = {
+    path: '/gochain-api/:command1?',
```
Comment (vikmeup) [inline mock/ext-api-dyson/get/gochain-api-commands1.js:11]: What is the purpose to test callisto-api-commands1.js  and gochain-api-commands1.js  is it's the same provider? it just creates tons of duplication

File: mock/ext-api-dyson/get/tomochain-api-commands1.js [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+                            ]
+                        }
+                    `);
+                }
+                break;        
+        }
+
+        // fallback
+        var return4Codacy = {error: "Not implemented"};
+        return return4Codacy;
```
Comment (vikmeup) [inline mock/ext-api-dyson/get/tomochain-api-commands1.js:89]:
```suggestion
        return {error: "Not implemented"};
```

File: mock/ext-api-dyson/get/callisto-api-commands1.js (hunk 1 above)
Comment (EnoRage) [reply mock/ext-api-dyson/get/callisto-api-commands1.js:18]: +
let it be separete PR, @catenocrypt what do you think?

Comment (catenocrypt) [reply mock/ext-api-dyson/get/callisto-api-commands1.js:18]: Discussed separately, will handle separately.

File: mock/ext-api-dyson/get/gochain-api-commands1.js (hunk 2 above)
Comment (catenocrypt) [reply mock/ext-api-dyson/get/gochain-api-commands1.js:11]: True, but end-to-end usecase is not the same even though internally it runs in the same implementation.

File: mock/ext-api-dyson/get/tomochain-api-commands1.js (hunk 3 above)
Comment (catenocrypt) [reply mock/ext-api-dyson/get/tomochain-api-commands1.js]: These were a desperate workaround to get around codacy, now it is solved with proper ignore, so I have removed these.

File: mock/ext-api-dyson/get/cosmos-api-commands4.js [hunk 4]
```diff
@@ -37,8 +37,7 @@ module.exports = {
         }
 
         // fallback
```
Comment (vikmeup) [inline mock/ext-api-dyson/get/cosmos-api-commands4.js:39]: do we need comment //fallback here?

Comment (catenocrypt) [reply mock/ext-api-dyson/get/cosmos-api-commands4.js:39]: Negligible value, but does it hurt?

Comment (vikmeup) [reply mock/ext-api-dyson/get/cosmos-api-commands4.js:39]: what's the purpose of comment?

Comment (catenocrypt) [reply mock/ext-api-dyson/get/cosmos-api-commands4.js:39]: Helps me understand why the Not implemented there when I revisit this code in 3 months and already forgot part of it.  It hints at the code structure of some combinations covered through some if/switch cases, and everything else falls through till the fallback.

Comment (vikmeup) [reply mock/ext-api-dyson/get/cosmos-api-commands4.js:39]: `return {error: "Not implemented"};` this is self explanatory. you can always go to history to see changes. I would avoid adding anything that's not needed

**Pull request merged**